src/core/globals.hpp

```diff
@@ -644,8 +644,9 @@ namespace big
 			bool keep_on_ground                         = false;
 			bool no_collision                           = false;
 			bool unlimited_weapons                      = false;
+			bool siren_mute				    = false;
```

Comment:
The entire section above uses spaces and you used tabs, you should probably run `clang-format`.

Or maybe it's just some new github UI bug, i don't now.

Reply:
Its a bug, if i use spaces it just puts it way ahead in github PR in my case